shell/platform/android/io/flutter/embedding/engine/android/AndroidTouchProcessor.java

```diff
@@ -269,6 +277,9 @@ private int getPointerChangeForAction(int maskedAction) {
     if (maskedAction == MotionEvent.ACTION_CANCEL) {
       return PointerChange.CANCEL;
     }
+    if (maskedAction == MotionEvent.ACTION_SCROLL) {
```

Comment:
What's the relationship between scrolling and hovering?  They seem to be showing up in pairs...

Reply:
The iOS embedding looked like it was encoding scroll events as (PointerChange=hover, PointerSignal=scroll), so I duplicated that.

fwiw, it looks like the `PointerEventConverter` logic in the framework will consider any event with PointerSignal=scroll to be a scroll event regardless of the PointerChange type.

Reply:
The intent is that discrete scroll events use hover, while trackpad gesture scrolling would use down/move/up sequences. Only the former is implemented in the framework currently, which is why it doesn't check the type, but in the future it will (unless that design changes).

Reply:
@stuartmorgan is that documented anywhere?  If not, can we document here or in an appropriate place that is referenced from here (not holding up this PR for that).

Reply:
It's not currently documented.

Documenting it sounds good; suggestions on an appropriate place? It seems like there should really be a document about pointer event streams in general, but if there is one I'm not sure where. (E.g., the only reason I know the distinction between `kMove` and `kHover` in general, or what `kAdd` and `kRemove` are for, is because I've read the converter code.) Perhaps this falls under the bug about documenting the embedding API in general?

Reply:
A thorough explanation in a Wiki doc would probably be good.  I would then try to provide surrounding context and basic explanation in the code, and then link to the full doc from those comments.  Generally speaking, I try to convey as much context in javadoc comments as possible because leaving the IDE for a website is a distraction if it can be avoided.